dev-libs/libsigc++/libsigc++-2.99.7.recipe

```diff
@@ -55,12 +54,13 @@ INSTALL()
 {
 	make install
 
-	prepareInstalledDevelLib libsigc-2.0
+	prepareInstalledDevelLib \
+		libsigc-3.0
```

Comment:
Comment about style: For prepareInstalledDevel**Libs** it's a good idea to split the line, but for prepareInstalledDevel**Lib** I think it is better to keep the library name on the same line as the command.